--- python-jose-3.2.0/setup.py.orig	2020-07-30 00:46:23.000000000 +0200
+++ python-jose-3.2.0/setup.py	2023-11-20 06:28:27.732542925 +0100
@@ -38,7 +38,7 @@ extras_require = {
     'pycrypto': ['pycrypto >=2.6.0, <2.7.0'] + pyasn1,
     'pycryptodome': ['pycryptodome >=3.3.1, <4.0.0'] + pyasn1,
 }
-legacy_backend_requires = ['ecdsa <0.15', 'rsa'] + pyasn1
+legacy_backend_requires = ['ecdsa', 'rsa'] + pyasn1
 install_requires = ['six <2.0']
 
 # TODO: work this into the extras selection instead.
